api/src/main/java/org/pmiops/workbench/config/StoredCredentialsConfig.java

```diff
@@ -0,0 +1,48 @@
+package org.pmiops.workbench.config;
+
+import com.google.auth.oauth2.ServiceAccountCredentials;
+import java.io.IOException;
+import org.pmiops.workbench.auth.Constants;
+import org.pmiops.workbench.google.CloudStorageService;
+import org.pmiops.workbench.google.CloudStorageServiceImpl;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.context.annotation.Lazy;
+
+@Configuration
+@Import(CloudStorageServiceImpl.class)
+public class StoredCredentialsConfig {
+  /**
+   * Service account credentials for Gsuite administration, corresponding to the "gsuite-admin"
+   * service account in each environment. Enabled for domain-wide delegation of authority.
+   */
+  @Lazy
+  @Bean(name = Constants.GSUITE_ADMIN_CREDS)
+  public ServiceAccountCredentials gsuiteAdminCredential(CloudStorageService cloudStorageService)
+      throws IOException {
+    return cloudStorageService.getGSuiteAdminCredentials();
```

Comment:
Probably just a TODO: this method (and friends) should not exist on cloudStorageService. Now that you have this bean, I would remove it there, and just have this call a basic readBlob() method instead; knowledge of that file's name etc would be better encoded in this provider.